site-src/index.md

```diff
@@ -54,10 +54,10 @@ resources), in the same cluster.
 
 ## Gateway API for Service Mesh (the [GAMMA initiative](/mesh/gamma)) <a name="for-service-mesh"></a>
 
-??? example "Standard in v1.2.0+"
+??? success "Standard Channel since v1.2.0"
```

Comment:
I'm not sure who introduced v1.2 here, but it wasn't you. I'll file a follow up PR to fix this.

Reply:
Agg, I fixed at least one of these. Sorry to miss this one.